src/helm/benchmark/metrics/basic_metrics.py

```diff
@@ -264,6 +264,34 @@ def bleu_1(gold: str, pred: str) -> float:
     return sentence_bleu([word_tokenize(gold)], word_tokenize(pred), weights=(1, 0, 0, 0))
 
 
+def chinese_bleu_1(gold: str, pred: str) -> float:
+    return sentence_bleu([[i for i in gold]], [i for i in pred], weights=(1, 0, 0, 0))
+
+
+def get_chinese_rouge_function(rouge_type: str) -> Callable[[str, str], float]:
+    class CharTokenizer:
+        def tokenize(self, text: str) -> List[str]:
+            return [char for char in text]
+
+    char_tokenizer = CharTokenizer()
```

Comment:
Chinese tokenizers are mandatory for BLEU and ROUGE scores because (almost) no empty space is provided in the string such that the default tokenizer can correctly split the text. We follow the idea of [sacrebleu](https://github.com/mjpost/sacrebleu/blob/bafca9df5b933770aa5c7fb7d858acd34ac43d4c/sacrebleu/tokenizers/tokenizer_zh.py#L93) in `CLEVAMachineTranslationMetric` and implement a Chinese character tokenizer here. For ROUGE score, this tokenizer is utilized by passing it to the `tokenizer` argument of the `__init__` function of `rouge_scorer.RougeScorer`. For BLEU, we directly split the Chinese string into a list of characters before feeding it to `sentence_bleu`.

Reply:
We made a bit of modification such that the Chinese tokenizer usage across metrics is clearer and unified.